.gitignore

```diff
@@ -3,6 +3,9 @@
 # dependencies
 /node_modules
 
+#config
+/src/config/config.json
+
```

Comment:
This needs to be there. Don't put it in .gitignore.  Only the code inside it needs changing by different users.

Reply:
Ok but I did not push the config.json file(with my own configuration) on github so do I need to push it as well ?